setup.cfg

```diff
@@ -0,0 +1,7 @@
+[versioneer]
+VCS = git
+style = pep440-branch-based
```

Comment:
@pelson How can we use this style of versioning if [python-versioneer PR90](https://github.com/warner/python-versioneer/pull/90) isn't merged yet?